packages/mattermost-components/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@mattermost/components",
+  "module": "dist/index.esm.js",
+  "types": "dist/index.esm.d.ts",
+  "scripts": {
+    "build": "rollup -c",
+    "debug": "rollup -c"
+  },
+  "workspaces": [
+    "../mattermost-redux",
```

Comment:
This is getting me thinking again about possibly shipping a minimal subset of mattermost-redux based on what we find we need here. I don't have any good solutions for that yet, but it'd be worth seeing what we need for these components and extracting that out. Perhaps that's even just the nebulous Suite API